guides/common/modules/proc_enabling-the-installer-managed-tftp-service.adoc

```diff
@@ -0,0 +1,17 @@
+[id="enabling-the-installer-managed-tftp-service"]
+= Enabling the installer-managed TFTP service
+
+If you do not have a TFTP server available in your network, you can use the installer-managed TFTP service to perform unattended installations.
+With the installer-managed TFTP service, you can run a TFTP server with a low maintenance effort because {Project} fully manages the TFTP service, including the files on that service.
+
+Perform the steps on the {Project} or {SmartProxyServer} that you want to configure as TFTP service.
```

Comment:
```suggestion
Perform the steps on the {Project} or {SmartProxyServer} that you want to configure to provide the TFTP service.
```
At least I think so?

Reply:
Or maybe you don't need this line at all. It seems to duplicate the step in the procedure.

Reply:
makes sense. I removed this sentence.